pandas/tests/frame/indexing/test_mask.py

```diff
@@ -99,3 +102,24 @@ def test_mask_try_cast_deprecated(frame_or_series):
     with tm.assert_produces_warning(FutureWarning):
         # try_cast keyword deprecated
         obj.mask(mask, -1, try_cast=True)
+
+
+def test_mask_stringdtype():
```

Comment:
hmm can you use frame_or_series here as well

Reply:
I added a test to tests/series/indexing/test_mask.py